deploy/webhook/member-operator-webhook.yaml

```diff
@@ -137,6 +145,30 @@ objects:
     namespaceSelector:
       matchLabels:
         toolchain.dev.openshift.com/provider: codeready-toolchain
+  - name: users.virtualmachines.webhook.sandbox
+    admissionReviewVersions:
+      - v1
+    clientConfig:
+      caBundle: ${CA_BUNDLE}
+      service:
+        name: member-operator-webhook
+        namespace: ${NAMESPACE}
+        path: "/mutate-virtual-machines"
+        port: 443
+    matchPolicy: Equivalent
+    rules:
+    - operations: ["CREATE"]
```

Comment:
The webhook's primary purpose is to ensure a VM will start when they try it in the sandbox for the first time so that's why I only have it for CREATE since the limits need to be defined the first time to ensure the VM can start.

The situation with UPDATE is a little less clear. In order for the webhook to do anything for the UPDATE case it would mean that the user has removed the limits from the VM after creation and maybe there's some reason why they'd want to do that. So that's why I didn't include it for UPDATE.

Reply:
Thanks for explaining that, so this means that users will be able to UPDATE `virtualmachines` resources after creation and the webhook will not interfere with the updates ?

Reply:
but don't we want to inject the SSH key in the future using the webhook as well? That would mean that user can remove it, which is something we don't want to let them do, right?

Reply:
just for the record, we discussed this offline and decided to keep it as only CREATE for now and we can add UPDATE later if necessary